nvmeadm/src/error.rs

```diff
@@ -34,6 +34,8 @@ pub enum NvmeError {
         host: String,
         port: u16,
     },
+    #[snafu(display("No Nvmf Subsystem found for nqn :{}", nqn))]
+    NoSubsytemFound { nqn: String },
```

Comment:
We already have an `NqnNotFound` error, could we reuse it?

Reply:
I thought that error is not descriptive enough. However, we dont print error anywhere so should be fine i guess.

Reply:
Incorporated

Reply:
Then you can fix that error and make it more descriptive, win win :)

Reply:
Oh I see that's what you did already, great! 👍